source/docs/advanced/features/smart-forms.md

```diff
@@ -51,6 +64,12 @@ module.exports = {
 };
 ```
 
+Restart the project, and your existing fields should now use Capency webservices to provide suggestions and validate data.
+
+<blockquote class="info">
+  **ProTip:** you can use the `front-commerce:smart-forms:capency` value in your `DEBUG` environment variable to view more details about API interactions with Capency services.
```

Comment:
this should also be mentioned in https://developers.front-commerce.com/docs/reference/environment-variables.html#Debugging (and maybe it's a good occasion that no other value was forgotten)

Reply:
Done in https://github.com/front-commerce/developers.front-commerce.com/pull/212